README.md

```diff
@@ -48,12 +52,6 @@ Local site is now running at `lvh.me:3000`
 
 Some other useful commands for developing can be found in the makefile.
 
-## Quickstart for everyone else
-
-### Set up Postgresql locally with docker
-
-If you don't already have Postgresql installed and running, you can set it up with docker, just run `make docker`. To tear down the docker setup, run `make clean`.
```

Comment:
I think you should leave this in? If you're not GFSC staff this is a simple way to get the DB up and running immediately